readconfig_test.go

```diff
@@ -44,4 +47,50 @@ func Test_ReadConfig(t *testing.T) {
 		t.Logf("Expected WriteDebug `%v` actual `%v`\n", true, config.WriteDebug)
 		t.Fail()
 	}
+
+	expectedMaxInflight := 10
+	if config.MaxInflight != expectedMaxInflight {
+		t.Logf("Expected maxInflight `%v` actual `%v`\n", expectedMaxInflight, config.MaxInflight)
+		t.Fail()
+	}
+
+	expectedAckWait := time.Millisecond * 10
+	if config.AckWait != expectedAckWait {
+		t.Logf("Expected maxInflight `%v` actual `%v`\n", expectedAckWait, config.AckWait)
+		t.Fail()
+	}
+
+	os.Unsetenv("max_inflight")
+	os.Unsetenv("ack_wait")
+
+	config = readConfig.Read()
+
+	expectedMaxInflight = 1
+	if config.MaxInflight != expectedMaxInflight {
+		t.Logf("Expected maxInflight `%v` actual `%v`\n", expectedMaxInflight, config.MaxInflight)
+		t.Fail()
+	}
+
+	expectedAckWait = time.Second * 30
+	if config.AckWait != expectedAckWait {
+		t.Logf("Expected maxInflight `%v` actual `%v`\n", expectedAckWait, config.AckWait)
+		t.Fail()
+	}
+
+	os.Setenv("max_inflight", "10.00")
+	os.Setenv("ack_wait", "10")
+
+	config = readConfig.Read()
+
+	expectedMaxInflight = 1
+	if config.MaxInflight != expectedMaxInflight {
+		t.Logf("Expected maxInflight `%v` actual `%v`\n", expectedMaxInflight, config.MaxInflight)
+		t.Fail()
+	}
+
+	expectedAckWait = time.Second * 30
+	if config.AckWait != expectedAckWait {
+		t.Logf("Expected maxInflight `%v` actual `%v`\n", expectedAckWait, config.AckWait)
```

Comment:
The language on this error and a couple others need to be updated. The message says "maxInFlight" but it should be "ackWait"

Reply:
@LucasRoesler my mistake, I will fixed it